src/fund/components/FundCardAmountInput.tsx

```diff
@@ -0,0 +1,171 @@
+      if (inputRef.current) {
+        inputRef.current.style.width = `${width}px`;
+        inputRef.current.style.maxWidth = `${390 - currencyWidth}px`;
+      }
+    }
+  }, [value]);
+
+  // biome-ignore lint/correctness/useExhaustiveDependencies: We only want to focus the input when the input type changes
+  useEffect(() => {
+    // focus the input when the input type changes
+    handleFocusInput();
+  }, [inputType]);
+
+  const handleFocusInput = () => {
+    if (inputRef.current) {
+      inputRef.current.focus();
+    }
+  };
+
+  return (
+    <div
+      className="flex cursor-text py-6"
+      onClick={handleFocusInput}
+      onKeyUp={handleFocusInput}
+    >
+      <style>
+        {`
+          input[type="number"]::-webkit-inner-spin-button,
+          input[type="number"]::-webkit-outer-spin-button {
+            -webkit-appearance: none;
+            appearance: none;
+            margin: 0;
+          }
+
+          input[type="number"] {
+            -moz-appearance: textfield;
+          }
+        `}
+      </style>
+
+      <div className="flex" style={{ height: '78px' }}>
+        {/* Display the fiat currency sign before the input*/}
+        {inputType === 'fiat' && currencySign && (
+          <FundCardCurrencyLabel
+            ref={currencySpanRef}
+            currencySign={currencySign}
+          />
+        )}
+
+        <input
+          className={cn(
+            text.body,
+            'border-none bg-transparent',
+            'text-[3.75rem] leading-none outline-none',
+          )}
+          type="number"
+          value={value}
+          onChange={handleChange}
+          ref={inputRef}
+          inputMode="decimal"
+          minLength={1}
+          placeholder="0"
+          data-testid="ockFundCardAmountInput"
+        />
+        {/* Display the crypto asset symbol after the input*/}
+        {inputType === 'crypto' && assetSymbol && (
+          <FundCardCurrencyLabel
+            ref={currencySpanRef}
+            currencySign={assetSymbol}
+          />
+        )}
+      </div>
+
+      {/* Hidden span for measuring text width 
+          Without this span the input field would not adjust its width based on the text width and would look like this:
+          [0.12--------Empty Space-------][ETH] - As you can see the currency symbol is far away from the inputed value
+
+          With this span we can measure the width of the text in the input field and set the width of the input field to match the text width
+          [0.12][ETH] - Now the currency symbol is displayed next to the input field
+      */}
+      <span
+        data-testid="ockHiddenSpan"
+        ref={hiddenSpanRef}
+        className={cn(
+          text.body,
+          'border-none bg-transparent',
+          'text-[3.75rem] leading-none outline-none',
+        )}
+        style={{
+          position: 'absolute',
+          opacity: 0,
+          whiteSpace: 'nowrap',
+          pointerEvents: 'none',
+        }}
```

Comment:
move into Tailwind utilities for consistency

Reply:
@dschlabach  Where is the Tailwing utilities? I can't find any in the codebase. (The css I see is autogenerated and I can't add anything there)

Reply:
All Tailwind utilities are accessible anywhere in /src. You can apply them directly as classes. https://tailwindcss.com/docs/utility-first